src/libs/actions/IOU.ts

```diff
@@ -5300,6 +5349,18 @@ function editRegularMoneyRequest(
         }
     }
 
+    // Update recently used currencies if the category is changed
```

Comment:
We might not even need this comment
```suggestion
    // Update recently used currencies if the currency is changed
```